src/test/java/org/sonarsource/sonarlint/ls/mediumtests/LanguageServerMediumTests.java

```diff
@@ -162,6 +163,26 @@ void analyzeSimplePythonFileOnOpen() throws Exception {
         tuple(1, 2, 1, 7, "python:PrintStatementUsage", "sonarlint", "Replace print statement by built-in function.", DiagnosticSeverity.Warning));
   }
 
+  @Test
+  void analyzePythonFileWithDuplicatedStringOnOpen() throws Exception {
+    String uri = getUri("analyzePythonFileWithDuplicatedStringOnOpen.py");
+
+    List<Diagnostic> diagnostics = didOpenAndWaitForDiagnostics(uri, "python", "def foo():\n  print('/toto')\n  print('/toto')\n  print('/toto')\n");
+
+    assertThat(diagnostics)
+      .extracting(startLine(), startCharacter(), endLine(), endCharacter(), code(), Diagnostic::getSource, Diagnostic::getMessage, Diagnostic::getSeverity)
+      .containsExactly(
+        tuple(1, 8, 1, 15, "python:S1192", "sonarlint", "Define a constant instead of duplicating this literal '/toto' 3 times. [+2 flows]", DiagnosticSeverity.Error));
```

Comment:
This rule is probably reporting multiple secondary locations, not multiple flows.